tests/stomach_contents_test.cpp

```diff
@@ -266,17 +267,14 @@ TEST_CASE( "vitamin_daily", "[vitamins]" )
             subject.get_daily_vitamin( vitamin_iron ) == 0 ) {
             break;
         }
-        //otherwise clean up any other health changes that may have happened
-        subject.set_daily_health( 0 );
-
     }
 
     // check if after a day health is up and vitamins are reset
     CHECK( subject.get_daily_vitamin( vitamin_vitC ) == 0 );
     CHECK( subject.get_daily_vitamin( vitamin_calcium ) == 0 );
     CHECK( subject.get_daily_vitamin( vitamin_iron ) == 0 );
     // get that vitamin health bonus is up by 6 with maybe a recent reduction on the same timecheck
-    CHECK( subject.get_daily_health() >= 5 );
+    CHECK( subject.get_lifestyle() >= 5 );
```

Comment:
```suggestion
    CHECK( subject.get_health_tally() >= 5 );
```

From my understanding all this test has to do is make sure the vitamins got processed, so this should be enough. Maybe also require `health_tally` to be 0 at the start of the test or check the increase instead of the absolute value.